pkg/app/ops/mysqlensurer/indexes.sql

```diff
@@ -1,6 +1,72 @@
--- TODO: Add all required indexes for PipeCD.
--- index on `updated_at` field on `Application` table in ASC direction
-CREATE INDEX application_updated_at_asc ON Application (updated_at ASC);
+--
+-- Application table indexes
+--
 
--- index on `created_at` field on `Application` table in DESC direction
-CREATE INDEX application_created_at_desc ON Application (created_at DESC);
+-- index on `Disabled` and `UpdatedAt` DESC
+CREATE INDEX application_disabled_updated_at_desc ON Application (Disabled, UpdatedAt DESC);
+
+-- index on `EnvId` ASC and `UpdatedAt` DESC
+ALTER TABLE Application ADD COLUMN EnvId VARCHAR(32) GENERATED ALWAYS AS (data->>"$.env_id");
```

Comment:
How about explicitly specifying the generated column to be `VIRTUAL`?

Reply:
I have no strong opinion on that, but you're right, explicitly specifying looks better 🙆‍♀️

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/1731/commits/044686ab18161ab59060a542bf5fc0ebb5791ad6